### PR TITLE
Fix #621: Create PowerAuth-2024.07 page

### LINKED ISSUE #621: Create PowerAuth-2024.07 page

### PR DESCRIPTION
**Pull request by banterCZ**

### CODE REVIEW

File: docs/PowerAuth-2024.07.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+| Component                   | Application Name            | Version | Description                                                                       |
+|-----------------------------|-----------------------------|---------|-----------------------------------------------------------------------------------|
+| PowerAuth Command Line Tool | `powerauth-java-cmd.jar`    | 1.8.0   | Command line tool for integration testing.                                        |
+| User Data Store             | `user-data-store.war`       | 1.3.0   | Server component which stores clients personal data securely.                     |
+| Mobile Utility Server       | `mobile-utility-server.war` | 1.8.0   | Server component for dynamic SSL pinning, text localization, and other utilities. |
+| SSL Pinning Tool            | `ssl-pinning-tool.jar`      | 1.8.0   | A command line utility used to sign SSL certificates for dynamic SSL pinning.     |
+
+
+### Mobile Libraries
```
Comment (banterCZ) [inline docs/PowerAuth-2024.07.md:40]: - [ ]  Update mobile libraries

@TomasKypta @Hopsaheysa @kober32 Could you please comment or even add suggestions to the PR?

**Review by romanstrobl — CHANGES_REQUESTED** [review]
Mobile SDK version and related components need to be updated, e.g.:
https://github.com/wultra/powerauth-mobile-sdk/releases/tag/1.8.1

File: docs/PowerAuth-2024.07.md (hunk 1 above)
Comment (TomasKypta) [reply docs/PowerAuth-2024.07.md:40]: I guess the mobile versions should be:
| PA SDK | `com.wultra.android.powerauth:powerauth-sdk` | 1.8.1 |
| React Native | `react-native-powerauth-mobile-sdk` | 2.5.1 |
| mToken SDK iOS | `WultraMobileTokenSDK` | 1.10.0 |
| mToken SDK Android | `com.wultra.android.mtokensdk:mtoken-sdk-android` | 1.11.0 |

File: docs/PowerAuth-2024.07.md [hunk 2]
```diff
@@ -41,13 +41,13 @@ For updating to 2024.07, please follow these migration guides:
...
 | watchOS            | `PowerAuth2ForWatch`                              | 1.8.0   | A limited library for watchOS.              |
 | iOS App Extensions | `PowerAuth2ForExtensions`                         | 1.8.0   | A limited library for iOS App Extensions.   |
-| Android            | `com.wultra.android.powerauth:powerauth-sdk`      | 1.8.0   | A client library for Android.               |
-| React Native       | `react-native-powerauth-mobile-sdk`               | 2.5.0   | React Native wrapper library for PowerAuth. | 
-| mToken SDK iOS     | `WultraMobileTokenSDK`                            | 1.9.0   | Mobile Token SDK for the iOS platform.      |
-| mToken SDK Android | `com.wultra.android.mtokensdk:mtoken-sdk-android` | 1.9.0   | Mobile Token SDK for the Android platform.  |
+| Android            | `com.wultra.android.powerauth:powerauth-sdk`      | 1.8.1   | A client library for Android.               |
+| React Native       | `react-native-powerauth-mobile-sdk`               | 2.5.1   | React Native wrapper library for PowerAuth. | 
+| mToken SDK iOS     | `WultraMobileTokenSDK`                            | 1.10.0  | Mobile Token SDK for the iOS platform.      |
```
Comment (TomasKypta) [inline docs/PowerAuth-2024.07.md:49]: 1.11.1

**Review by romanstrobl — APPROVED** [review]
Looks OK.

**Review by TomasKypta — APPROVED** [review]

**Pull request merged**